web/gui/dashboard_info.js

```diff
@@ -3951,19 +3951,19 @@ netdataDashboard.context = {
             ],
         },
         */        
-        info: 'Percentage of returned/fetched rows.'
+        info: 'Percentage of fetched vs returned rows. This metric measures the percentage of rows that contain data needed to execute the query, out of the total number of rows scanned. A high value indicates that the database is executing queries efficiently, while a low value indicates that the database is performing extra work by scanning a large number of rows that aren\'t required to process the query. Low values may be caused by missing indexes or inefficient queries.'
```

Comment:
```suggestion
        info: 'The percentage of rows that contain data needed to execute the query, out of the total number of rows scanned. A high value indicates that the database is executing queries efficiently, while a low value indicates that the database is performing extra work by scanning a large number of rows that aren\'t required to process the query. Low values may be caused by missing indexes or inefficient queries.'
```

How about this? I think if there is a way to shorten the text (without losing information), we should always do so.